docs/reference/operations/create_transfers.md

```diff
@@ -0,0 +1,309 @@
+valid account id.
+
+### `credit_account_id_must_not_be_zero`
+The transfer was not created.
+[`Transfer.credit_account_id`](../transfers.md#credit_account_id) is zero, but must be a valid
+account id.
+
+### `credit_account_id_must_not_be_int_max`
+The transfer was not created.
+[`Transfer.credit_account_id`](../transfers.md#credit_account_id) is `2^128 - 1`, but must be a
+valid account id.
+
+### `accounts_must_be_different`
+The transfer was not created.
+[`Transfer.debit_account_id`](../transfers.md#debit_account_id) and
+[`Transfer.credit_account_id`](../transfers.md#credit_account_id) must not be equal.
+
+That is, an account cannot transfer money to itself.
+
+### `pending_id_must_be_zero`
+The transfer was not created.
+Only post/void transfers can reference a pending transfer.
+
+Either:
+- [`Transfer.flags.post_pending_transfer`](../transfers.md#flagspost_pending_transfer) must be set, or
+- [`Transfer.flags.void_pending_transfer`](../transfers.md#flagsvoid_pending_transfer) must be set, or
+- [`Transfer.pending_id`](../transfers.md#pending_id) must be zero.
+
+### `pending_transfer_must_timeout`
+The transfer was not created.
+[Transfer.timeout](../transfers.md#timeout) is zero, but must nonzero for pending transfers.
+
+### `ledger_must_not_be_zero`
+The transfer was not created.
+[`Transfer.ledger`](../transfers.md#ledger) is zero, but must be nonzero.
+
+### `code_must_not_be_zero`
+The transfer was not created.
+[`Transfer.code`](../transfers.md#code) is zero, but must be nonzero.
+
+### `amount_must_not_be_zero`
+The transfer was not created.
+[`Transfer.amount`](../transfers.md#amount) is zero, but must be nonzero.
+
+Every transfer must move value.
+Only posting and voiding transfer amounts may be zero — when zero, they will move the full pending
+amount.
+
+### `debit_account_not_found`
+The transfer was not created.
+[`Transfer.debit_account_id`](../transfers.md#debit_account_id) must refer to an existing `Account`.
+
+### `credit_account_not_found`
+The transfer was not created.
+[`Transfer.credit_account_id`](../transfers.md#credit_account_id) must refer to an existing `Account`.
+
+### `accounts_must_have_the_same_ledger`
+The transfer was not created. The accounts referred to by
+[`Transfer.debit_account_id`](../transfers.md#debit_account_id) and
+[`Transfer.credit_account_id`](../transfers.md#credit_account_id) must have an identical
+[`ledger`](../accounts.md#ledger).
+
+(TODO: Link to explanation of implementing currency exchange with liquidity accounts).
+
+### `transfer_must_have_the_same_ledger_as_accounts`
+The transfer was not created. The accounts referred to by
+[`Transfer.debit_account_id`](../transfers.md#debit_account_id) and
+[`Transfer.credit_account_id`](../transfers.md#credit_account_id) are equivalent, but
+differ from the [`Transfer.ledger`](../transfers.md#ledger).
+
+### `exists_with_different_flags`
+The transfer already exists, but with different [`flags`](../transfers.md#flags).
```

Comment:
Clarifying here as well, by transfer already exists we are just talking about the transfer's id right?

Reply:
Yes, I'll clarify that.